### DISCUSSION TITLE
Where can I find the raw data?

### DISCUSSION THREAD

**Discussion by jiangzihanict** [deepmodeling/deepmd-kit#337] [Q&A]
The raw data seems to be preprocessed into .npy format and four sets in the [water example](https://github.com/deepmodeling/deepmd-kit/tree/master/examples/water/data). I can find the description of the raw data in the [readme](https://github.com/deepmodeling/deepmd-kit/blob/devel/doc/use-deepmd-kit.md#prepare-data), but no specific rawdata file is found.  Could I get the raw data and divide it into more and smaller sets? And is there a possible way to get more raw data?

Thanks!

**Comment by amcadmus** [comment]
Hi 
the .npy data is standard numpy binary you can load them by numpy.load.

It is recommended to use [dpdata](https://github.com/deepmodeling/dpdata) to convert data in other formats to deepmd supported compressed format, including the conversion from compressed data to raw data and vise versa.

User should be responsible for the data generation.

**Reply by jiangzihanict** [reply]
Got it. Thanks for the quick reply.
